src/App.css

```diff
@@ -1,38 +1,63 @@
-.App {
-  text-align: center;
+.container {
+  align-items: center;
+  padding: 0px;
+  background: #181818;
+}
+.topbar {
+  width: 1097px;
+  height: 119px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  display: flex;
+  flex-direction: row;
+  margin: auto;
+  justify-content: space-between;
+  align-items: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.menu {
+  align-items: center;
+  padding: 0px;
+  gap: 10px;
+  color: #fff;
+  width: 393px;
+  height: 44px;
```

Comment:
Remove the height and width here and make sure to add padding to each text accordingly ao that there won't be a fixed width